src/renderer/components/dex/OrderHistory.vue

```diff
@@ -45,6 +45,10 @@
         </table>
       </div>
     </div>
+    <div class="footer">
```

Comment:
May want to see what it would look like in the header, but aligned to the right; new mocks will likely have it there to conserve space.

Reply:
I'll update when we get mocks.